.github/workflows/cli-release.yaml

```diff
@@ -0,0 +1,68 @@
+# This defines a workflow to build and release a new version of the aptos CLI.
+# In order to trigger it go to the Actions Tab of the Repo, click "Release CLI" and then "Run Workflow".
+
+name: "Release CLI"
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_tag:
+        type: string
+        required: true
+        description: "The release tag to create. E.g. v0.2.3:"
+      source_git_ref_override:
+        type: string
+        required: false
+        description: "GIT_SHA1_OVERRIDE: Use this to override the Git SHA1, branch or tag to build the binaries from. Defaults to the workflow Git REV, but can be different than that:"
```

Comment:
So, does this only release on a tag e.g. `aptos-cli-v0.1.1`?

Reply:
> So, does this only release on a tag e.g. `aptos-cli-v0.1.1`?

Nope. The way this works is you as the "admin" have to go to the Actions tab and trigger the workflow manually. There will be displayed a mandatory field into which you have to enter a tag name, such as `aptos-cli-v0.1.1` . You can technically enter any tag name. Note that GitHub actions will create the Tag for you as part of the workflow (instead of you creating the Tag directly via Git).